.env

```diff
@@ -1,6 +1,11 @@
 # Misc
 APP_DIR=/usr/src/app
 
+# Db
```

Comment:
@mitjade Could you use `DB` instead of `Db` everywhere?

Reply:
done